maat-court-data-api/src/main/java/gov/uk/courtdata/dto/OutstandingAssessmentResultDTO.java

```diff
@@ -0,0 +1,16 @@
+package gov.uk.courtdata.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class OutstandingAssessmentResultDTO {
+    @Builder.Default
+    boolean outstandingAssessments = false;
+    String message;
+}
```

Comment:
along with the default `outstandingAssessments ` value as false, I think it would be nice to have a default message as well in the response in place of null. 
`{
    "outstandingAssessments": false,
    "message": null
}`

Reply:
I have added a Jackson annotation to exclude empty fields as the message field won't be used by the consumer of this API when there are no outstanding assessments